packages/cli/src/CommandDescriptor.ts

```diff
@@ -165,10 +166,10 @@ export const map: {
  * @since 1.0.0
  * @category combinators
  */
-export const mapOrFail: {
+export const mapEffect: {
   <A, B>(f: (a: A) => Either<ValidationError, B>): (self: Command<A>) => Command<B>
```

Comment:
Need to toggle the types here

Reply:
Thanks for that - I knew I missed something <3